testing/efl/APKBUILD

```diff
@@ -13,55 +13,74 @@ depends_dev="giflib-dev gstreamer-dev gst-plugins-base-dev libpng-dev
 	libxscrnsaver-dev xorgproto luajit-dev"
 makedepends="$depends_dev dbus-dev openssl-dev fontconfig-dev freetype-dev bash
 	fribidi-dev zlib-dev"
+depends=""
+depends_dev="luajit-dev"
+makedepends="$depends_dev
```

Comment:
makedepends and depends_dev is already defined two lines earlier, please don't define it twice but append to the previous variables instead.

Reply:
I can really be a moron sometimes. Sorry

Reply:
No worries